server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/delivery/LocalDelivery.java

```diff
@@ -119,7 +119,6 @@ public void init(MailetConfig mailetConfig) throws MessagingException {
             .resourceLocator(ResourceLocatorImpl.instanciate(usersRepository, sieveRepository))
             .consume(getInitParameter("consume", true))
             .deliveryHeader("Delivered-To")
-            .resetReturnPath(true)
```

Comment:
do you mean that there's no configuration entry for that ? Would it make sense to have one ?